c2corg_ui/templates/helpers/view.html

```diff
@@ -397,11 +397,11 @@ <h3 class="heading closed show-phone" ng-click="mainCtrl.animateHeaderIcon($even
 
 
 ## Float buttons
-<%def name="show_float_buttons(outingUsers, doctype, doc_id, doc_lang, other_langs, missing_langs)">\
+<%def name="show_float_buttons(associatedUsers, doctype, doc_id, doc_lang, other_langs, missing_langs, image_type = 'default')">\
```

Comment:
I feel uncomfortable with using this ``default`` image type since it actually does not exist.
I would rather make it ``collaborative`` by default. Then your test below
```
ng-if="'${image_type}' == 'collaborative' || userCtrl.hasEditRights(${dumps(associatedUsers)})"
```
would become
```
% if image_type != 'collaborative' :
    ng-if="userCtrl.hasEditRights(${dumps(associatedUsers)})"
% endif
```